ERROR - 2023-02-24 11:06:31 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'vntech'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('vntech.mysql.uh...', 'vntech', '@Vntech2023', 'vntech', 3306, '', 0)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-02-24 11:06:31 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'45.178.113.37' (using password: YES
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `menu_home`
WHERE `menu_home`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-02-24 11:18:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'vntech'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('vntech.mysql.uh...', 'vntech', '@Vntech2023', 'vntech', 3306, '', 0)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-02-24 11:18:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'45.178.113.37' (using password: YES
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `menu_home`
WHERE `menu_home`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-02-24 14:38:02 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'vntech'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('vntech.mysql.uh...', 'vntech', '@Vntech2023', 'vntech', 3306, '', 0)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-02-24 14:38:02 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'45.178.113.37' (using password: YES
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `menu_home`
WHERE `menu_home`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-02-24 17:11:43 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'vntech'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('vntech.mysql.uh...', 'vntech', '@Vntech2023', 'vntech', 3306, '', 0)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-02-24 17:11:43 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'45.178.113.37' (using password: YES
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `menu_home`
WHERE `menu_home`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-02-24 17:11:57 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'vntech'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('vntech.mysql.uh...', 'vntech', '@Vntech2023', 'vntech', 3306, '', 0)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-02-24 17:11:57 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'45.178.113.37' (using password: YES
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `menu_home`
WHERE `menu_home`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-02-24 17:12:10 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'vntech'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('vntech.mysql.uh...', 'vntech', '@Vntech2023', 'vntech', 3306, '', 0)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-02-24 17:12:10 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'45.178.113.37' (using password: YES
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `menu_home`
WHERE `menu_home`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-02-24 17:15:01 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'vntech'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('vntech.mysql.uh...', 'vntech', '@Vntech2023', 'vntech', 3306, '', 0)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-02-24 17:15:01 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'45.178.113.37' (using password: YES
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `menu_home`
WHERE `menu_home`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-02-24 17:34:34 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'vntech'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('vntech.mysql.uh...', 'vntech', '@Vntech2023', 'vntech', 3306, '', 0)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-02-24 17:34:34 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'45.178.113.37' (using password: YES
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `menu_home`
WHERE `menu_home`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-02-24 17:40:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'vntech'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('vntech.mysql.uh...', 'vntech', '@Vntech2023', 'vntech', 3306, '', 0)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-02-24 17:40:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'45.178.113.37' (using password: YES
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `menu_home`
WHERE `menu_home`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-02-24 17:40:15 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'vntech'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('vntech.mysql.uh...', 'vntech', '@Vntech2023', 'vntech', 3306, '', 0)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'45.178.113.37' (using password: YES in C:\xampp\htdocs\site\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\site\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\site\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\site\system\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\site\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\site\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\site\app\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\site\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\site\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\site\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\site\web\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-02-24 17:40:15 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'45.178.113.37' (using password: YES
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1601): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `menu_home`
WHERE `menu_home`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
